### PR TITLE
flatpak: add screenshots to cockpit-client metainfo

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
These are required for flathub.

### CODE REVIEW

File: containers/flatpak/cockpit-client.metainfo.xml.in [hunk 1]
```diff
@@ -18,6 +18,21 @@
     </p>
   </description>
 
+  <screenshots>
+    <screenshot>
+      <caption>Cockpit Client login screen</caption>
+      <image>http://cockpit-project.org/images/flatpak/flatpak-screenshot-login.png</image>
```
Comment (martinpitt) [inline containers/flatpak/cockpit-client.metainfo.xml.in:24]: https s'il te plaît -- they are redirected to https anyway, so let's save the user the extra redirection.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Rupert!

File: containers/flatpak/cockpit-client.metainfo.xml.in (hunk 1 above)
Comment (allisonkarlitskaya) [reply containers/flatpak/cockpit-client.metainfo.xml.in:24]: oops.  thanks!

**Review by martinpitt — APPROVED** [review]
*so* much moar secuh-re now :wink:

**Pull request merged**